src/stories/ExternalSubtitlesTemplate/ExternalSubtitlesTemplate.js

```diff
@@ -0,0 +1,18 @@
+import React, { useState } from "react";
+import EkoVideoTemplate from "../EkoVideoTemplate/EkoVideoTemplate";
+import { EkoSubtitles } from '../../components/EkoSubtitles/EkoSubtitles';
+
+export default function ExternalSubtitlesTemplate(args, context){
+    let [playerRef, setPlayerRef] = useState();
+    args.onPlayerInit = (player) => { setPlayerRef(player) };
+    let ekoVideo = EkoVideoTemplate(args, context);
+    let style = args.style || {};
+    return (
+        <div style={{height: 100 + '%', width: 100 + '%'}}>
+            <div style={{height: 80 + '%'}}>
+                {ekoVideo}
+            </div>
+            <EkoSubtitles player={playerRef} style={style}/>
+        </div>
```

Comment:
I completely missed where are the subtitles in the storybook example (especially since they're not present from the start)
How about 
1. Implementing a simple design to make it clearer in the story:
![image](https://user-images.githubusercontent.com/3951311/106522223-08565d00-64e8-11eb-9bb7-191859dc39ac.png)

Sample output code (with ugly inline style):
```
<h5 style="
    padding: 10px 30px;
    margin: 0;
    color: white;
    background: black;
">External Subtitles below:</h5>
<div role="Subtitles" style="
    background: black;
    color: white;
    padding: 10px 30px;
">
    
<div class="eko_subtitles_div "><span>[Upbeat background music]</span></div></div>
```

2. Have the video pause at a certain time when you know for a certain there are subtitles to be displayed in the selected example video. This would also help the visual tests, which now don't contain any actual subtitles:

![image](https://user-images.githubusercontent.com/3951311/106522454-71d66b80-64e8-11eb-9523-50871f722ded.png)

You can look at `EkoVideoTemplate.js` for an example on how to do that

Reply:
My ExternalSubtitlesTemplate actually uses the EkoVideoTemplate, so I just added a way to set a specific seek time. Let me know if you'd prefer to handle this differently!